extensions/ql-vscode/package.json

```diff
@@ -2081,6 +2081,7 @@
   "dependencies": {
     "@octokit/plugin-retry": "^6.0.1",
     "@octokit/rest": "^20.0.2",
+    "@types/imurmurhash": "^0.1.2",
```

Comment:
If we use MurmurHash, can we also declare a dependency on `imurmurhash`? Right now, the only reason we have access to it because it's included in one of our dev dependencies (`eslint`). It would be unexpected that if eslint removes their use of the package, our code would suddenly break.